### ISSUE TITLE
Make a proper README

### ISSUE THREAD

**Issue by tokee** [ukwa/webarchive-discovery#250]
The real documentation is on the wiki, but the README is where people land when they see the project on GitHub, so it also needs to hold enough to explain what this is all about. Currently it is dominated by the Elasticsearch documentation and besides that says very little. It should be expanded quite a bit.

**Comment by anjackson** [comment]
Good idea.  Should we move the Solr Quickstart stuff into the README? Or move the ES stuff into the wiki?

I found some templates that help me think about what could be included:

- https://gist.github.com/PurpleBooth/109311bb0361f32d87a2
- https://github.com/othneildrew/Best-README-Template
